docs/campaign/quests/substrate-pallets.md

```diff
@@ -17,32 +17,159 @@ If you are unsure which chain state function to call, you can also use this step
 7. If the user follows the space, you'll receive a `true` response. Otherwise, you'll get `false`.
 
 ### Configuring the task on AirLyft
 
-AirLyft simplifies the process of creating Substrate Pallet tasks, making it as straightforward as using Polkadotjs. Here's how to configure this task:
+- In your campaign page, go to the "Quests" tab, and select the **Substrate Pallets** quest from the options.
+  ![Substrate Pallets Quest on AirLyft](../../images/pallets-3.png)
+
+- You will then be presented with options for selecting Substrate Pallet actions. Let's read in depth about each of them.
+
+  ![](../../images/palletsOptions.png)
+
+- [Check Substrate Balance](#check-substrate-balance)
+- [Check Substrate Stake amount](#check-substrate-stake-amount)
+- [Check Substrate AssetHub Asset Balance](#check-substrate-assethub-asset-balance)
+- [Check If Substrate AssetHub Hold NFT](#allow-only-specific-accounts-on-twitter)
+- [Substrate Raw Querry](#substrate-raw-querry)
+- [Substrate Bond to Pool](#check-if-account-bonded-to-pool)
+- [Substrate Nominate Validator](#check-if-account-has-nominated-validator)
+
+AirLyft simplifies the process of creating Substrate Pallet tasks, making it as straightforward as using Polkadotjs. Let's dive deep into it.
+
+#### Check Substrate Balance
+
+- This option allows you to verify whether a user has a minimum required balance of a specific token in their Substrate-based wallet. This is particularly useful for campaigns that reward users for holding or maintaining a certain balance of an asset, ensuring they have an active presence in the ecosystem before participating in a quest.
+
+1. Select the Substrate Balance option, as shown in the screenshot below.
+
+   ![](../../images/palletsOptions.png)
+
+2. Enter basic information such as the **Quest Name, a brief Description, the cFuel allocation, and XP rewards**. This information helps define the purpose of the quest and the incentives for completing it.
+
+   ![](../../images/palletAssetHubAssetBalance.png)
+
+3. To configure the **Task Condition & Recurrence**, refer to the [Task Condition & Recurrence](../task-condition-and-recurrence.md) page.
+
+4. Choose the **Blockchain** where the balance should be checked (e.g., Polkadot AssetHub, Subsocial, etc.). Enter the **Minimum balance** that users must hold to be eligible for completing the quest. This ensures that only users meeting the criteria can proceed.
+
+5. Once all details are filled in, click the **Add Task** button to complete the process. The Substrate Balance quest is now set up and ready for users to participate.
+
+#### Check Substrate Stake Amount
+
+- Staking plays a vital role in securing blockchain networks, and this option enables you to check if a user has staked a specific amount of tokens. By using this task, you can create campaigns that incentivize users to participate in staking, ensuring they are actively contributing to network security before earning rewards.
```

Comment:
`Staking plays a vital role in securing blockchain networks,` not needed

Reply:
`ensuring they are actively contributing to network security before earning rewards.` not needed